src/main/resources/liquibase/changelogs/2023/10/19-01.yaml

```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1697693652731-1
```

Comment:
would be better use more understandable id in here for example: Created categories table (also you can add time)